pgrx-tests/src/framework.rs

```diff
@@ -678,8 +777,18 @@ pub(crate) fn get_pg_dbname() -> &'static str {
 }
 
 pub(crate) fn get_pg_user() -> String {
-    std::env::var("USER")
-        .unwrap_or_else(|_| panic!("USER environment var is unset or invalid UTF-8"))
+    get_runas().unwrap_or_else(|| {
+        std::env::var("USER")
+            .unwrap_or_else(|_| panic!("USER environment var is unset or invalid UTF-8"))
+    })
+}
+
+fn get_runas() -> Option<String> {
+    std::env::var("CARGO_PGRX_TEST_RUNAS").ok()
```

Comment:
`VarError` has two cases. `VarError::NotPresent` and `VarError::NotUnicode`. Please either make this function return OsString (and so, use `env::var_os`, probably), or panic on encountering `VarError::NotUnicode`.

Also, this is a string, but we're probably assuming something about the shape of the contents. For instance, is whitespace allowed? Should it not be? How about `"$VARIABLE"`? That... *should* be harmless, but

Reply:
Fixed error handling.

GIGO I think is fine.  Because of the `Command` type, we'll at least be passing whatever the user provides to `--runas` through as-is as a proper argument.  iow, we don't have to worry about shell escaping or anything like that.